Django_web/myfirst/urls.py

```diff
@@ -0,0 +1,24 @@
+"""myfirst URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/3.2/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import path, include
+
+urlpatterns = [
+    path('articles/', include('articles.urls')),

```

Comment:
In the case of opening index page, we have got 404. 
![image](https://user-images.githubusercontent.com/6921551/126362020-798fc557-8b46-4ce2-9fcd-436f83db7ea3.png)
It is not user-friendly for any site :) Could we fix it?

Reply:
Yes, changed urls and added page template :)
![image](https://user-images.githubusercontent.com/63068458/126717665-6de60c50-8121-491a-a0d1-15e30ad10122.png)